build/yaml/jobs/windows/postbuild.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Job Template: PostBuild steps
+
+jobs:
+- job: PostBuild
+  displayName: PostBuild
+
+  variables:
+    Packaging.EnableSBOMSigning: true
+
+  pool:
+    name: VSEngSS-MicroBuild2019-1ES
+    demands:
```

Comment:
I don't think any of these demands are necessary.

Reply:
removing